packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -45,7 +45,13 @@ describeCompat("Message size", "NoCompat", (getTestObjectProvider, apis) => {
 	beforeEach("getTestObjectProvider", () => {
 		provider = getTestObjectProvider();
 	});
-	afterEach(async () => provider.reset());
+	afterEach(async function () {
+		// This test is consistently failing when ran against FRS. See ADO:7925
+		if (provider.driver.type === "routerlicious" && provider.driver.endpointName === "frs") {
+			this.skip();
+		}
```

Comment:
This seems incorrect, if there are any tests that are still running against frs we need the afterEach hook to still apply to them. I'd also say, I'd expect that if a test is failing in the afterEach hook, it's probably timing out, not a for-sure real failure (like an assert failing) and in that case I'd say skipping it might not be the right solution.